unlock-app/src/utils/postOffice.ts

```diff
@@ -92,25 +92,7 @@ export function setupPostOffice<T extends MessageTypes = MessageTypes>(
     const listeners = handlers[event.data.type]
     if (listeners && listeners.size) {
       listeners.forEach(listener => {
-        if (debug) {
```

Comment:
All of these `if (debug)` statements counted against our coverage, but they were primarily a development aid when the current paywall was implemented. They are not necessary at this time, and even less so since we're moving into a new paywall architecture entirely.